proto/cosmos/group/v1beta1/testdata_tx.proto

```diff
@@ -0,0 +1,12 @@
+syntax = "proto3";
+
+package cosmos.group.v1beta1;
+
+import "gogoproto/gogo.proto";
+
+option go_package = "github.com/cosmos/cosmos-sdk/x/group";
+
+// MsgAuthenticated is used in group module's tests to check msg authorizations
+message MsgAuthenticated {
```

Comment:
where is this message used?

Can we use the SDK's https://github.com/cosmos/cosmos-sdk/blob/45cad1df0ae0557fa22bbc5d97a7474329cc7443/testutil/testdata/tx.proto#L23-L28 ?